e-text-field-type.yml

```diff
@@ -1630,3 +1658,32 @@ components:
         - 'CITIZENSHIP_OF_FIRST_PERSON'
         - 'CITIZENSHIP_OF_SECOND_PERSON'
         - 'CVV'
+        - 'DATE_OF_INSURANCE_EXPIRY'
+        - 'MORTGAGE_BY'
+        - 'OLD_DOCUMENT_NUMBER'
+        - 'OLD_DATE_OF_ISSUE'
+        - 'OLD_PLACE_OF_ISSUE'
+        - 'DLCLASSCODE_LR_FROM'
```

Comment:
let's name it  DL_CLASSCODE_** (see other DL examples)
DL - driver license (also, add description as other DL classes)